library/core/src/future/future.rs

```diff
@@ -23,6 +24,7 @@ use crate::task::{Context, Poll};
 /// When using a future, you generally won't call `poll` directly, but instead
 /// `.await` the value.
 ///
+/// [`async`]: ../../core/keyword.async.html
```

Comment:
```suggestion
/// [`async`]: ../../keyword.async.html
```

Reply:
And somewhat same for others based on the error. I recommend you to test locally, it'll allow you to avoid some more "push and wait".